docs-guides/source/flexible-inputs.md

```diff
@@ -168,15 +168,15 @@ If the source PyTorch model is exported by [`torch.export.export`](https://pytor
 
 ## Reshape Frequency Optimization Hint
 
-Setting the Reshape Frequency Optimization Hint to `Frequent` can allow flexible shaped models to run on the Neural Engine. This option can be set when loading your model:
+Setting the Reshape Frequency Optimization Hint to `Infrequent` can allow flexible shaped models to run on the Neural Engine, with iOS 17.4 or later.
+This option can be set when loading your model:
```

Comment:
Maybe also point to the doc on developer.apple.com, so that devs know how to use it in their apps.

Reply:
I considered that when I originally wrote these docstrings, but decided against it. I felt [that documentation](https://developer.apple.com/documentation/coreml/mloptimizationhints-swift.struct/reshapefrequency-swift.enum) didn't meaningfully add any information. Also, we generally don't share such links in our documentation.